README.md

```diff
@@ -192,3 +193,23 @@ workflows:
                 - master
           run_group_id: "123"
 ```
+
+## Orb Release Process
+This section describes the release process for the orb itself:
+1. Create a feature branch and do your work
+1. Update the version in the repo: `git grep -z -l -F 1.1.0 | xargs -0 sed -i -e 's/1.1.0/1.2.0/g'`
```

Comment:
Did you do this? :D

Reply:
No, because the change doesn't have any semver impact.